src/views/product-tutorials-view/server.ts

```diff
@@ -57,37 +41,33 @@ export interface ProductPageData {
 export async function getCloudTutorialsViewProps() {
 	const productData = cachedGetProductData('hcp')
 
+	/**
+	 * Fetch and process the authored page content
+	 */
+	const { pageData, headings } = await getProcessedPageData(ThemeOption.cloud)
+
 	/**
 	 * Build the sidebar
 	 */
 	const hcpCollections = await getCollectionsBySection('cloud')
 	const sidebarSections = buildCategorizedHcpSidebar(hcpCollections)
 
 	/**
-	 * Get the raw page data
+	 * Build sitemap collections, if we're using them.
 	 */
-	const {
-		pageData: rawPageData,
-		inlineCollections,
-		inlineTutorials,
-	} = await getProductPageContent(ThemeOption.cloud)
+	let sitemapCollections: SitemapCollection[]
```

Comment:
💭 Should we default this to an empty array?

Reply:
It's probably not necessary, since these will get defined if the show sitemap boolean is true. Feel free to ignore

Reply:
I'll leave as-is for now 👍